.github/workflows/.test.yml

```diff
@@ -365,112 +365,3 @@ jobs:
         name: Show matrix
         run: |
           echo ${{ steps.tests.outputs.matrix }}
-
-  integration-cli:
```

Comment:
Do they still fail?

Reply:
@AkihiroSuda I added them back and they don't fail.

I removed them to speed-up the CI run with several tests (bisect, etc.). But it makes sense to add them back once I got a result, to verify the result